lib/cmd/fh3/admin/logs/syslogs.js

```diff
@@ -0,0 +1,71 @@
+/* globals i18n */
+var common = require('../../../../common.js');
+var _ = require('underscore');
+
+module.exports = {
+  'desc': i18n._('Search for log entries associated with a request ID'),
+  'examples': [{
+    cmd: 'fhc admin syslogs --requestId="requestid12345" --projects="core,mbaas"',
+    desc: i18n._('Searching for log entries associated with request ID: requestid12345')
+  }],
+  'demand': ['requestId', 'projects'],
+  'describe': {
+    'requestId': i18n._("A request ID to search for."),
+    'projects': i18n._("A comma-separated list of project names to search for logs in."),
+    'numLogEntries': i18n._("The maximum number of log entries returned. Default is 1000."),
+    'order': i18n._("The order of entries to be displayed. [asc, desc]")
```

Comment:
Should probably make it explicit that this ordering is by time, and maybe even rename it to something along the lines of 'recent first/last'